src/components/Article.jsx

```diff
@@ -0,0 +1,42 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import "../App.css";
+import { getArticleById } from "../../Utils/api";
+
+const Article = () => {
+  const { article_id } = useParams();
+
+  const [newArticle, setNewArticle] = useState({});
+  const [isloading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    getArticleById(article_id)
+      .then((dataFromApi) => {
+        setNewArticle(dataFromApi);
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        setIsError(true);
+      });
+  }, []);
+
+  if (isloading) {
+    return <p>Loading...</p>;
+  }
+
+  return (
+    <div className="singleArticle">
+      <h1>{newArticle.title}</h1>
+      <h2>Topic: {newArticle.topic} </h2>
+      <img src={newArticle.article_img_url} />
+      <p>
+        Item details <br />
```

Comment:
This is a bit of a strange title for the article and also isn't very distinguishable from the text - can you reconsider/ reformat this to make it a bit clearer?